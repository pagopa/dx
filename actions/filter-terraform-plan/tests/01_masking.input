PRIVATE_KEY = "-----BEGIN PRIVATE KEY-----MIIBIjANBgkqhki...-----END PRIVATE KEY-----"
