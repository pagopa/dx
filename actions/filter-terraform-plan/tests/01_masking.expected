Test mode enabled.
--- Executing Plan ---
PRIVATE_KEY = "[REDACTED]"
